organizing_hub/views/call_views.py

```diff
@@ -402,6 +461,59 @@ def post(self, request, *args, **kwargs):
         """Return CSV"""
         return response
 
+class CallCampaignUpdateView(
+    LocalGroupPermissionRequiredMixin,
+    SuccessMessageMixin,
+    UpdateView
+):
+    template_name = "calls/callcampaign_form.html"
+    form_class = CallCampaignUpdateForm
+    model = CallCampaign
+    organizing_hub_feature = OrganizingHubFeature.calling_tool
+    permission_required = 'calls.change_callcampaign'
+    slug_field = 'uuid'
+    slug_url_kwarg = 'uuid'
+    success_message = '''
+    Your calling campaign has been edited succesfully.
+    '''
+
+    def form_valid(self, form):
+        caller_emails = form.cleaned_data['caller_emails']
+        caller_ids = get_or_create_callers(caller_emails)
+        form.instance.callers = caller_ids
+        return super(CallCampaignUpdateView, self).form_valid(form)
```

Comment:
Nice 👍  So this will both add and remove callers?

Reply:
Yes.